helper/validation.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+function image_check($pic){
```

Comment:
Add function header and also rename the function with proper name like validate_image or image_validation